install-and-configure/install/multi-cluster/secondary-clusters.md

```diff
@@ -32,16 +32,10 @@ Grafana is not needed on secondary clusters.
 
 Kubecost and its accompanying Prometheus collect a reduced set of metrics that allow for lower resource/storage usage than a standard Prometheus deployment.
 
-The following configuration options further reduce resource consumption when not using the Kubecost frontend:
-
 ```
 --set prometheus.server.retention=2d
```

Comment:
Is there a reason we should leave this helm value here?

Reply:
Although the default Prometheus retention is now 97h ([code ref](https://github.com/kubecost/cost-analyzer-helm-chart/blob/b128331ceaabb44e5bc4b2b1d3fd3dbe4abca52e/cost-analyzer/values.yaml#L1269-L1294)), I think it's still worth pointing out that this can be further tuned if needed! What do you think?

Reply:
The default retention value is now `97h` (~4 days) so this recommendation is a reduction of half.